### PR TITLE
SG-21170-ROUND3 [ZD to AKN] Update links referenced from code to the old support site to point to AKN

### PR DESCRIPTION
**Pull request by 000paradox000**
Update URLs for documentation and toolkit general information.

### CODE REVIEW

**Review by rob-aitchison — DISMISSED** [review]
👍🏻

File: README.md [hunk 1]
```diff
@@ -5,8 +5,8 @@
 This repository is a part of the ShotGrid Pipeline Toolkit.
 
 - For more information about this app and for release notes, *see the wiki section*.
-- For general information and documentation, click here: https://support.shotgunsoftware.com/entries/95441257
-- For information about ShotGrid in general, click here: https://help.autodesk.com/view/SGSUB/ENU
+- For general information and documentation, click here: https://developer.shotgridsoftware.com/d587be80/?title=Integrations+User+Guide
+- For information about ShotGrid in general, click here: https://developer.shotgridsoftware.com/162eaa4b/?title=Pipeline+Integration+Components
```
Comment (mathurf) [inline README.md:9]: This mention information about ShotGrid in general, not about Toolkit's apps and engines. I think previous url was good

**Review by mathurf — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (000paradox000) [reply README.md]: Done

**Review by mathurf — APPROVED** [review]
LGTM!
:shipit:

**Pull request merged**